.gitattributes

```diff
@@ -1,4 +1,30 @@
 docs/cloud/index.md linguist-generated=true
+docs/cloud/how-to-create-a-ticket-for-temporal-support.md linguist-generated=true
```

Comment:
This is authored directly in `docs/cloud`. We might want to revisit that decision.